docs/tutorials/quickstart.md

```diff
@@ -4,10 +4,17 @@ Blueapi acts as a worker that can run bluesky plans against devices for a specif
 laboratory setup. It can control devices to collect data and export events to tell
 downstream services about the data it has collected.
 
-## Start ActiveMQ
+## Start RabbitMQ
 
-The worker requires a running instance of ActiveMQ, the simplest
-way to start it is to run it via a container:
+The worker requires a running instance of RabbitMQ. The easiest way to start it is
+ to execute the provided script:
+
+```
+    src/script/start_rabbitmq.sh
```

Comment:
Must: This fails with the error `Error: context must be a directory: "/blueapi/rabbitmq_setup"`. I think the script only works if your current directory is `blueapi/src/scripts`. Can we allow it to be run from the top please?